src/style.css

```diff
@@ -1 +1,70 @@
 /* Styles go here */
+* {
+  box-sizing: border-box;
+  text-decoration: none;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+  font-size: 12px;
+}
+
+header {
+  display: flex;
+  background-color: #fff;
+  justify-content: space-between;
+}
+
+nav {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-right: 50px;
+  text-transform: uppercase;
+}
+
+.logo {
+  align-self: center;
+  margin-left: 50px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  padding-left: 0;
+  margin: 0;
+}
+
+.nav__link {
+  position: relative;
+  display: inline-block;
+  color: #000;
+  margin: 0 10px;
+  padding: 23px 0;
+}
+
+.is-active {
+  color: #00acdc;
+  margin-left: 0;
+}
+
+.lastOne {
+  margin-right: 0;
+}
+
+.is-active::after {
+  display: block;
+  content: "";
+  position: absolute;
+  bottom: 0;
+  width: 36px;
```

Comment:
Don't use fixed width. Use `left: 0` and `right: 0` so it would work with any link width.